reporting_endpoint.py

```diff
@@ -169,6 +169,14 @@ def unclaimed_proposal_payment(table1, table2, start_date, end_date):
     result = utils.int2asset(result)
     return result
 
+def authorized_unclaimed_payment_proposal(table1, table2):
```

Comment:
function that checks the proposal status and payment status.
used to see whether the amount claimable is higher that the amount in the ops wallet.